examples/get-stream-settings/src/main.rs

```diff
@@ -0,0 +1,34 @@
+// example for reading stream settings
+
+use arctic::H10MeasurementType;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let mut polar = arctic::PolarSensor::new("9F8F7521".to_string())
```

Comment:
If cli-app has an ID input I'd prefer this to have one as well.

Reply:
Ok